### ISSUE TITLE
Allow canceling draft event

### ISSUE THREAD

**Issue by ercasta** [CoderDojo/community-platform#1110]
Currently it's not possible to cancel an event when it's in draft (the button does not appear). To cancel an event, it's necessary to publish it and only then the "cancel" button appears

**Comment by Wardormeur** [comment]
Wouldn't it makes more sense to have a "delete" when it's a draft ?

**Comment by ercasta** [comment]
Yes, probably "delete" is more appropriate... but the point is there is no button right now. It only appears after the event is published

**Comment by Wardormeur** [comment]
Because it was probably intended that Cancel "Cancel" the event, not dismiss changes. But it does dismiss changes atm.
This is very confusing, I agree :p I'll have a look early next week to come with a standrd. However, i'm not a big fan of having 4 buttons ( ideas @DanielBrierton ?)
